ext/lib/crypto/Kconfig

```diff
@@ -7,6 +7,5 @@
 menu "Cryptography"
 
 source "ext/lib/crypto/tinycrypt/Kconfig"
-source "ext/lib/crypto/mbedtls/Kconfig"
```

Comment:
Do we need to keep this in the tree for dependency management? I don't understand the conclusion of that discussion about module kconfigs.

Reply:
it is in modules/Kconfig.mbedtls